libs/iota-browser/src/validators/error.ts

```diff
@@ -0,0 +1,34 @@
+import { ErrorEvent, ErrorEventSchema } from '../validators/events'
+
+export enum ErrorCode {
+  'SIGNED_REQUEST_EVENT' = 'SignedRequestEvent',
+  'SIGNED_REQUEST_JWT' = 'SignedRequestJWT',
+  'RESPONSE_CALLBACK_EVENT' = 'ResponseCallbackEvent',
+  'VERIFIABLE_PRESENTATION_SCHEMA' = 'VerifiablePresentationSchema',
+  'ERROR_EVENT' = 'ErrorEvent',
+}
+
+function getIssue(errorEvent: ErrorEvent) {
+  return errorEvent.error.details![0].issue &&
+    errorEvent.error.details![0].issue.length > 0
+    ? errorEvent.error.details![0].issue
+    : errorEvent.error.message
+}
```

Comment:
isn't this error prone. I'm not sure how we can assume with the `!` that details always exist ?

Reply:
If details won't exist, it'll take error.message which will exist and that is expected behaviour.